cli/admin/README.md

```diff
@@ -126,5 +126,69 @@ Note that if you are using the standalone deployment, Bookkeeper commands (and o
 available. For this reason, we encourage you to go a step further and deploy a full Pravega cluster to 
 explore the whole functionality of Pravega Admin CLI.
 
+## Pravega CLI on Kubernetes
+
+The Pravega Admin CLI needs to be executed from inside the Kubernetes cluster, as it requires to reach the services that form a Pravega cluster.
+
+To this end, you can either spark a new pod in the cluster and build the tool, or log in one of the Pravega server pods that already has the Pravega Admin CLI built in. From now on, we will describe how to use the Pravega Admin CLI in Kubernetes assuming the latter option.
```

Comment:
conventional language: "spark" -> "start"

grammar: "log in one" -> "log in to one"

suggestion for last sentence: "In this section, we will assume the latter option and describe how to use the Pravega Admin CLI from within existing Pravega server pods."

Reply:
Re: "and build the tool" -- Consider explaining how to download the CLI tools via a release tarball or zip file. Users of official releases should be able to download the tools rather than build the tools.

@RaulGracia -- Re: download vs. build -- `pravega-0.9.0.{tgz,zip}` includes server and CLI. Would a CLI-only tarball/zip artifact provide any convenience to users? Download size might be the only benefit. Just a thought!

Reply:
@derekm yes, at the moment both user and admin CLIs are packaged together with Pravega server. It may make sense to have a separate artifact for CLI alone, but it is something that we could discuss for 0.10.